apps/yggdrasil/lib/yggdrasil/role_resource.ex

```diff
@@ -0,0 +1,13 @@
+defmodule Yggdrasil.RoleResource do
+  use Ecto.Schema
+  import Ecto.Changeset
+
+  schema "roles_resources" do
+    belongs_to :role, Yggdrasil.Role
+    belongs_to :resource, Yggdrasil.Resource
+
+    has_many :role_resource_permissions, Yggdrasil.RoleResourcePermission
```

Comment:
Hmm so yeah good question. Thinking a little to hierchal I suppose. 

One table simplifies thing so yeah good catch. I don't think there will be a need to show resource by it self without permissions so yeah cool with just one table.